### PR TITLE
Add unit tests for tv

### PR DESCRIPTION
**Pull request by KazuCocoa**
https://github.com/appium/appium-xcuitest-driver/issues/913

---

I've confirmed ruby_lib_core tvOS test case passed

### CODE REVIEW

File: WebDriverAgentLib/Utilities/FBTVNavigationTracker.h [hunk 1]
```diff
@@ -43,6 +54,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (FBTVDirection)directionToFocusedElement;
 
+/**
+ Define for testing: horizontalDirectionWithItem, verticalDirectionWithItem
+ */
+- (FBTVDirection)horizontalDirectionWithItem:(FBTVNavigationItem *)item andDelta:(CGFloat)delta;
```
Comment (mykola-mokhnach) [inline WebDriverAgentLib/Utilities/FBTVNavigationTracker.h]: such methods can be defined separately  in `-Private` header file

**Comment by KazuCocoa** [comment]
worked 👌 
https://travis-ci.org/appium/WebDriverAgent/jobs/533087925

**Comment by imurchie** [comment]
🥇

File: WebDriverAgentTests/UnitTests_tvOS/Doubles/XCUIElementDOuble.m [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/**
```
Comment (mykola-mokhnach) [inline WebDriverAgentTests/UnitTests_tvOS/Doubles/XCUIElementDOuble.m]: Please fix the file name (`DOuble`)

File: WebDriverAgentTests/UnitTests_tvOS/Doubles/XCUIElementDouble.h [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+@property (nonatomic, readwrite, assign) CGRect wdFrame;
+@property (nonatomic, readwrite, copy, nonnull) NSString *wdUID;
+@property (nonatomic, copy, readwrite, nullable) NSString *wdName;
+@property (nonatomic, copy, readwrite, nullable) NSString *wdLabel;
+@property (nonatomic, copy, readwrite, nonnull) NSString *wdType;
+@property (nonatomic, strong, readwrite, nullable) NSString *wdValue;
+@property (nonatomic, readwrite, getter=isWDEnabled) BOOL wdEnabled;
+@property (nonatomic, readwrite, getter=isWDVisible) BOOL wdVisible;
+@property (nonatomic, readwrite, getter=isWDAccessible) BOOL wdAccessible;
+#if TARGET_OS_TV
```
Comment (mykola-mokhnach) [inline WebDriverAgentTests/UnitTests_tvOS/Doubles/XCUIElementDouble.h]: does it make sense to have such if here? I assume there tests will be exclusively executed for this platform

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**